src/Eccube/Controller/Admin/Product/ProductController.php

```diff
@@ -980,4 +981,56 @@ private function createProductCategory($Product, $Category, $count)
 
         return $ProductCategory;
     }
+
+    /**
+     * Bulk public action
+     *
+     * @Route("/%eccube_admin_route%/product/bulk/product-status/{id}", requirements={"id" = "\d+"}, name="admin_product_bulk_product_status")
+     *
+     * @param Request $request
+     * @param integer $id
+     *
+     * @return RedirectResponse
+     */
+    public function bulkProductStatus(Request $request, $id)
+    {
+        if ($request->getMethod() != Request::METHOD_POST) {
+            throw new BadRequestHttpException('Unsupported method!');
+        }
+
+        $this->isTokenValid();
+
+        /** @var ProductStatus $ProductStatus */
+        $ProductStatus = $this->productStatusRepository->find($id);
```

Comment:
L1003 & L1004 is not need.
because, argument changed to `ProductStatus $ProductStatus`.